integration-cli/docker_cli_run_test.go

```diff
@@ -4010,6 +4011,20 @@ func (s *DockerSuite) TestRunNamedVolumesFromNotRemoved(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunAttachFailedNoLeak(c *check.C) {
+	// TODO @msabansal - https://github.com/moby/moby/issues/35023. Duplicate
+	// port mappings are not errored out on RS3 builds. Temporarily disabling
+	// this test pending further investigation. Note we parse kernel.GetKernelVersion
+	// rather than system.GetOSVersion as test binaries aren't manifested, so would
+	// otherwise report build 9200.
+	if runtime.GOOS == "windows" {
+		v, err := kernel.GetKernelVersion()
```

Comment:
Probably unlikely to fail, but check err anyway, or assign to `_` instead?

Reply:
I don't think there's any harm here having the check. But Atoi is highly unlikely to fail is GetKernelVersion succeeds

Reply:
Yeah, just seems a little weird to assign to `err` and not use the value.

Reply:
Oh, no it is used. See next line :)

Reply:
Sorry, yes, of course, looks good. :)